redis/commands/graph/query_result.py

```diff
@@ -360,3 +360,166 @@ def cached_execution(self):
     @property
     def run_time_ms(self):
         return self._get_stat(INTERNAL_EXECUTION_TIME)
+
+
+class AsyncQueryResult(QueryResult):
+    def __init__(self):
+        pass
+
+    async def initialize(self, graph, response, profile=False):
```

Comment:
docstring + type hints

Reply:
Type hints will be added in a separate PR.